docs/DeveloperGuide.md

```diff
@@ -288,44 +288,78 @@ _{Explain here how the data archiving feature will be implemented}_
 
 Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unlikely to have) - `*`
 
-| Priority | As a …​                                    | I want to …​                 | So that I can…​                                                        |
-|----------|--------------------------------------------|------------------------------|------------------------------------------------------------------------|
-| `* * *`  | new user                                   | see usage instructions       | refer to instructions when I forget how to use the App                 |
-| `* * *`  | user                                       | add a new person             |                                                                        |
-| `* * *`  | user                                       | delete a person              | remove entries that I no longer need                                   |
-| `* * *`  | user                                       | find a person by name        | locate details of persons without having to go through the entire list |
-| `* *`    | user                                       | hide private contact details | minimize chance of someone else seeing them by accident                |
-| `*`      | user with many persons in the address book | sort persons by name         | locate a person easily                                                 |
+| Priority | As a …​                                    | I want to …​           | So that I can…​                                                        |
+|----------|--------------------------------------------|------------------------|------------------------------------------------------------------------|
+| `* * *`  | user                                       | add detail of company  | easily keep track of companies i am interested in                      |
+| `* * *`  | user                                       | list companies added   | view all the companies that i am currently interested in               |
+| `* * *`  | user                                       | delete a company       | remove company that i am no longer interested in                       |
+| `* * *`  | user                                       | save my data           | view my list from previous sessions without reentering my list         |
+| `* *`    | user                                       | find a company by name | locate details of company without having to go through the entire list |
+| `* *`    | user with many persons in the address book | sort companies         | view the list in my preferred order                                    |
```

Comment:
i think this could be "user with many companies in the address book" since we are tracking companies but other than that everything LGTM

Reply:
I agree with @jasperng-nus on this! It would make the purpose of the address book very clear.

Reply:
Resolved